### ISSUE TITLE
Improve contrast of colors

### ISSUE THREAD

**Issue by g00glen00b** [g00glen00b/gatsby-blog#39]
**Description**:

Colors that fail the WCAG AA recommendation:

- The "new" indicator
- The category text
- All normal links
- All buttons

**Comment by g00glen00b** [comment]
Fixed with fe8cb999a5400c0622cf6044dafe33ddeeed7292
